Source/Client/Sync/SyncHandlers.cs

```diff
@@ -302,7 +302,7 @@ static void RenameZone(Dialog_RenameZone __instance)
         [MpPrefix(typeof(TransferableUIUtility), "DoCountAdjustInterface")]
         static void TransferableAdjustTo(Transferable trad)
```

Comment:
Consider decorating this method with the `[UsedImplicitly]` attribute.

Reply:
See my earlier comment on UsedImplicitly. Also, barely touched this existing method.